app/controllers/admin/communities_controller.rb

```diff
@@ -308,7 +309,8 @@ def update_settings
        MarketplaceService::API::Api.configurations.update({
         community_id: @current_community.id,
         main_search: params[:main_search],
-        distance_unit: params[:distance_unit]
+        distance_unit: params[:distance_unit],
+        limit_search_distance: params[:limit_distance].present?
```

Comment:
A checkbox parameter is either `"true"` (string) or `nil`, so `.present?` maps that nicely to boolean